KinConnections/app.py

```diff
@@ -31,5 +34,32 @@ def home():
     else:
         return redirect(url_for("login", error=error))
 
+
+@app.route('/send_email/<id>', methods=['GET', 'POST'])
```

Comment:
Let's get rid of this route in here since you've already moved over all the code to its own class